sklearn/multioutput.py

```diff
@@ -607,6 +625,8 @@ class ClassifierChain(MetaEstimatorMixin, ClassifierMixin, _BaseChain):
         Pass an int for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
 
+    verbose : bool, optional (default=False)
```

Comment:
```suggestion
    verbose : bool, default=False
```
Following the current [documentation guidelines](https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation)